btcclient/query.go

```diff
@@ -19,6 +19,10 @@ func (c *Client) GetBestBlock() (uint32, error) {
 		return 0, err
 	}
 
+	if height < 0 || height > int64(^uint32(0)) {
```

Comment:
what does the char ^ does in `int64(^uint32(0))` ?

Reply:
It's a bitwise NOT operator, flips all bits of `uint32(0)` effectively giving max uint32 value

Reply:
nice, very clever!